### ISSUE TITLE
2.76b Fails to run on Mac

### ISSUE THREAD

**Issue by bheiland** [zeffii/Blender_CAD_utils#32]
Trying to install the add-on on 2.76b and the add-on does not launch .  I get a Traceback message.

**Comment by zeffii** [comment]
install from this zip then: https://github.com/zeffii/mesh_tinyCAD/archive/master.zip  
if that also doesnt work please take a screenshot of the error and paste it here.

**Comment by bheiland** [comment]
This is what I see on the screen when I try to enable.

**Comment by zeffii** [comment]
cool. thanks for the report. will fix asap. I'll spare you the detail of why that happens.

You will have to delete the folder  `mesh_tinyCAD-master`  from the scripts/addons folder. redownload in an 30 minutes or so. should work fine.

**Comment by zeffii** [comment]
![image](https://cloud.githubusercontent.com/assets/619340/12300754/2ae0469a-ba1d-11e5-9609-da3b72381843.png)

**Comment by zeffii** [comment]
that's fixed now. you will have to download again.

I'll document the bug anyway, you skip reading this if you aren't interested in the code-reason why this happened.

Github generates zips with "addon_name" and appens the word "-master" to them, this makes zips called "addon_name-master". My code however had some reliance on the name being just "addon_name" or in my case "mesh_tinyCAD" (without the postfix "-master").

So in code I have a few lines at the top of 3 of for of these files that say 

    from mesh_tinyCAD import cad_module as cm

but because the folder isn't called mesh_tinyCAD it doesn't find the cad_module. However, if I write it like this 

    from . import cad_module as cm

the `period` stands for the parent directory. and that means I don't have to hard code the word "mesh_tinyCAD" anywhere.

Locally (on my machine) I don't need to download from github, so have never encountered this. I think most people that use the addon probably get a more elaborate blender zip from https://builder.blender.org/download/ which contains a lot of extra addons including this addon. (with the correct folder name).

TLDR;

thanks for the bug report, I didn't realize I still have code like that.

**Comment by bheiland** [comment]
I was able to get this version working but I had to manually move the files in the add ons directory.  They initially installed in a directory tree one level too deep.  I would have to remove and reinstall to test it again.

**Comment by zeffii** [comment]
no need to download again then. but it's fixed for the future.
